viz/pkg/healthcheck/healthcheck.go

```diff
@@ -69,22 +69,55 @@ func (hc *HealthChecker) RunChecks(observer healthcheck.CheckObserver) (bool, bo
 
 // VizCategory returns a healthcheck.Category containing checkers
 // to verify the health of viz components
```

Comment:
We should add a comment describing what `fullCheck` does.

Reply:
Done